qoqo/build.rs

```diff
@@ -117,6 +117,22 @@ impl<'ast> Visit<'ast> for Visitor {
                     return;
                 }
             }
+
+            // TEMP: REMOVE WHEN STABILISED
+            if matches!(att.style, AttrStyle::Outer)
+                && path == Some("cfg".to_string())
+                && !cfg!(feature = "unstable_simulation_repetitions")
+            {
+                let cfg_feature_name: CfgFeatureMacroArgument =
+                    att.parse_args().expect("parsing failed 1");
+                if cfg_feature_name
+                    .0
+                    .contains("unstable_simulation_repetitions")
+                {
+                    return;
+                }
+            }
+
```

Comment:
with the proper OR conditions I think we can keep this to a single if, together with what we have above already

Reply:
I'm going to leave two separate if blocks because it's easier to remove one when the corresponding operation is stabilized